pkg/messaging/messaging.go

```diff
@@ -32,21 +40,49 @@ for each finding in your console.
 	
 All your finding data stays only with us at Code Intelligence
 and will never be shared.`,
-			additionalParams: url.Values{
-				"utm_source":   []string{"cli"},
-				"utm_campaign": []string{"login-message"},
-				"utm_term":     []string{"b"},
+				additionalParams: url.Values{
+					"utm_source":   []string{"cli"},
+					"utm_campaign": []string{"run-message"},
+					"utm_term":     []string{"b"},
+				},
+			},
+		},
+		"Finding": {
+			{
+				message: fmt.Sprintf(`Authenticate with CI Fuzz %s 
+to get more insights on your findings and persist them for a full history.`, server),
+				additionalParams: url.Values{
+					"utm_source":   []string{"cli"},
+					"utm_campaign": []string{"finding-message"},
+					"utm_term":     []string{"a"}},
+			},
+			{
+				message: `With a free authentication you receive detailed information such as severity
+and solution tips for each finding in your console.
+	
+All your finding data stays only with us at Code Intelligence and will never be shared.`,
+				additionalParams: url.Values{
+					"utm_source":   []string{"cli"},
+					"utm_campaign": []string{"finding-message"},
+					"utm_term":     []string{"b"},
+				},
 			},
 		},
 	}
 
+	messageAndParam, entryPresent := messagesAndParams[string(context)]
```

Comment:
... then we also don't need the cast here:

```suggestion
	messageAndParam, entryPresent := messagesAndParams[context]
```